tracer/test/test-applications/instrumentation/CallTargetNativeTest/Program.cs

```diff
@@ -301,6 +316,15 @@ static void RunTests(string[] args)
                         // GenericParentAbstractMethod();
                         break;
                     }
+                case "remove":
+                    {
+                        WithOutArguments();
+                        DisableDefinitions();
+                        WithOutArguments(false);
+                        EnableDefinitions();
+                        WithOutArguments();
+                        break;
+                    }
                 default:
                     Console.WriteLine("Run with the profiler and use a number from 0-9/withref/without/abstract/all as an argument.");
```

Comment:
Could you add `remove` to this list and also copy/paste the method calls from the `remove` case to the `all` case?

```suggestion
                    Console.WriteLine("Run with the profiler and use a number from 0-9/withref/without/abstract/remove/all as an argument.");
```

Reply:
Fixed in 0b2b4beb398e906cd57bdeb185d06765f46791ae